src/Apps/Order/redirects.tsx

```diff
@@ -209,64 +209,54 @@ export const newCheckoutEnabled = ({
   )
 }
 
-export const newDetailsEnabled = ({
-  order,
-  featureFlags,
-}: Order2RedirectArgs): boolean => {
-  return !!(
-    (order.mode === "BUY" || order.mode === "OFFER") &&
-    featureFlags?.isEnabled?.("emerald_order-details-page")
```

Comment:
Should we archive this feature flag in Unleash?

Reply:
Yes!